ee/fe/modules/500-openvpn/openapi/config-values.yaml

```diff
@@ -205,3 +205,5 @@ properties:
           format: int64
         value:
           type: string
+  storageClass:
+    type: string
```

Comment:
The `description` field is missing. This field will be shown in the docs, so we need to provide a description for it and mention that this field is deprecated.

Reply:
Wanted to do without it. Added a description.